src/plugins/footnotes/index.ts

```diff
@@ -234,6 +225,47 @@ const deleteFootnoteWidget =
  *
  */
 
+export const buildPluginState = (doc: ManuscriptNode): PluginState => {
+  const inlineFootnotes: [InlineFootnoteNode, number][] = []
+  const footnotes: Map<string, [FootnoteNode, number]> = new Map()
+  let footnoteElement: [FootnotesElementNode, number] | undefined
+
+  doc.descendants((node, pos, parentNode) => {
+    if (isFootnoteNode(node)) {
```

Comment:
This condition may not be enough, as it will also include table footnotes. Maybe you should also check if it has a parent `footnotes_section`.

Reply:
Thanks, I've updated that